### PR TITLE
WIP: workshops sorted by weight, added to every post

### PR DESCRIPTION
**Pull request by trabaldini**
Every post has a weight now so it's possible to sort them as wanted

### CODE REVIEW

**Review by marcoow — CHANGES_REQUESTED** [review]
The weight approach is good but I think we should auto-assign a weight based on the workshop title in the update script (otherwise we have to manually add the weights again every time we update the workshops). I'd make it so that we assign a weight of 1 for all workshops that have _"ember"_ in the title, 2 for all that have _"elixir"_ or _"phoenix"_ in the title and 3 for all others (make sure that the match is case-insensitive).

File: _config.yml [hunk 1]
```diff
@@ -45,12 +45,11 @@ facebook:
...
-      image: 
-        twitter: /images/apple-144.png
-        facebook: /images/apple-144.png
-        height: 144
-        width: 144
+    image: 
+    twitter: /images/apple-144.png
+    facebook: /images/apple-144.png
+    height: 144
+    width: 144
```
Comment (marcoow) [inline _config.yml]: These changes seem unrelated?

**Review by marcoow — COMMENTED** [review]
I think it would be easier to do this in the [update script](https://github.com/simplabs/simplabs.github.io/blob/master/script/update-workshops#L29) so that we don't have to add logic to the templates:

```rb
weight = if title =~ /ember/i
  1
else if title =~ /elixir|erlang|phoenix/
  2
else
  3
end

course_yaml = {
  layout:      'workshop',
  title:       title,
  weight: weight,
  permalink:   "/training/#{slug}",
  category:    category,
  description: description,
  image:       "/#{image_name}"
}
```

**Comment by trabaldini** [comment]
closes #74

**Review by marcoow — APPROVED** [review]

**Pull request merged**